oat-sa/tao-test-runner-qti-fe
# feature: TR-199 custom tabstops order for bottom toolbar

Related to https://oat-sa.atlassian.net/browse/TR-199

- add custom keyboard navigation to bottom bar

Required: https://github.com/oat-sa/extension-tao-nfer/pull/245

How to check:
 - create delivery with more than one test item and REBL theme enabled
 - check the focusable elements order is according to task
